tests/plugins/test_deepspeed_plugin.py

```diff
@@ -361,6 +361,36 @@ def test_deepspeed_custom_activation_checkpointing_params(tmpdir):
     assert checkpoint_config["synchronize_checkpoint_boundary"]
 
 
+@RunIf(min_gpus=1, deepspeed=True)
```

Comment:
```suggestion
@RunIf(min_gpus=1, deepspeed=True, standalone=True)
```

Since deepspeed actually initializes processes, we need to treat this test as standalone. We might be able to combine this with other tests later on.